### PR TITLE
Final fixes in project

### PR DESCRIPTION
**Pull request by LucasBruzzone**
This PR was one after a thorough search in the whole project for final fixes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -128,26 +125,26 @@ In order to create a certificate we need to create a .cer file is a security fil
 ## Navigation
 There are several examples of different types of navigations in different branches
 #### Branch master
-Uses a Stack Navigator [documentation](https://reactnavigation.org/docs/stack-navigator/#props).
+Uses a Stack Navigator. [documentation](https://reactnavigation.org/docs/stack-navigator/#props).
```
Comment (lilhand) [inline README.md:128]: You added a period but there was already one at the end, are both necessary?

File: README.md [hunk 2]
```diff
@@ -183,11 +178,12 @@ You will notice both OS already have a Splash Screen set, so you just need to ch
 ## Change Icons
 
 ### Android
-go to `android/app/src/main/res` and drag your set of folders with the new icons, you should have a set of `mipmap` folders, one for each resolution.
+go to `android/app/src/main/res` and drag your set of folders with the new icons, you should have a set of `mipmap` folders, one for each resolution.
```
Comment (lilhand) [inline README.md:181]: Blank space after line.

File: README.md [hunk 3]
```diff
@@ -128,26 +125,26 @@ In order to create a certificate we need to create a .cer file is a security fil
 ## Navigation
 There are several examples of different types of navigations in different branches
 #### Branch master
-Uses a Stack Navigator [documentation](https://reactnavigation.org/docs/stack-navigator/#props).
+Uses a Stack Navigator. [documentation](https://reactnavigation.org/docs/stack-navigator/#props).
 Uses an Auth-first navigation flow.
 
 #### Branch master--optional-auth
-Uses a simple Stack Navigator [documentation](https://reactnavigation.org/docs/stack-navigator/#props).
+Uses a simple Stack Navigator. [documentation](https://reactnavigation.org/docs/stack-navigator/#props).
```
Comment (lilhand) [inline README.md:132]: Same.

File: README.md [hunk 4]
```diff
@@ -128,26 +125,26 @@ In order to create a certificate we need to create a .cer file is a security fil
...
 Lets you navigate through the app without being authed, but certain features need to be accessed with auth.
 As an example of an authed screen you can see ScreenThree.
 
 ### Nested Navigators
 
 #### Branch nested-tab-in-stack
-Uses a Tab Navigator nested in a Stack Navigator [documentation](https://reactnavigation.org/docs/bottom-tab-navigator) [documentation] (https://reactnavigation.org/docs/nesting-navigators/).
+Uses a Tab Navigator nested in a Stack Navigator. [documentation](https://reactnavigation.org/docs/bottom-tab-navigator)
```
Comment (lilhand) [inline README.md:139]: Here there are no periods.

File: README.md [hunk 5]
```diff
@@ -128,26 +125,26 @@ In order to create a certificate we need to create a .cer file is a security fil
...
 Uses an Auth-first navigation flow.
 
 #### Branch nested-tab-in-stack--optional-auth
-Uses a Tab Navigator nested in a Stack Navigator [documentation](https://reactnavigation.org/docs/bottom-tab-navigator) [documentation] (https://reactnavigation.org/docs/nesting-navigators/).
+Uses a Tab Navigator nested in a Stack Navigator [documentation](https://reactnavigation.org/docs/bottom-tab-navigator)
```
Comment (lilhand) [inline README.md:143]: Same here.

**Review by ldelarco — APPROVED** [review]

**Review by ldelarco — APPROVED** [review]

**Pull request merged**